web/admin/src/Admin.js

```diff
@@ -307,11 +298,17 @@ export default class Admin extends PureComponent {
           const leaderboard = Object.keys(activeSession.scores)
             .filter(userId => users[userId])
             .sort(sortUsers) // Sort by descending score and break ties with ascending avg answer time
-            .map(userId => ({
-              score: data.val().scores[userId].score,
-              firstName: users[userId].firstName,
-              lastName: users[userId].lastName,
-              email: users[userId].email,
+            .map((userId, i) => ({
+              Place: i + 1,
+              Score: data.val().scores[userId].score,
```

Comment:
We already have `data.val()` stored in `activeSession` so no need to repeatedly call the `val()` function.